boa3/internal/model/type/classes/classtype.py

```diff
@@ -118,6 +118,16 @@ def instance_methods(self):
             instance_funcs.update(base.instance_methods)
         return instance_funcs
 
+    @property
+    def methods(self):
+        """
+        :rtype: Dict[str, boa3.internal.model.method.Method]
+        """
```

Comment:
I don't understand why the return type needs to be written as a docstring instead of something like:
```python
from boa3.internal.model.method import Method
    @property
    def methods(self) -> Dict[str, Method]:
        pass
```

This happens a lot in this file, is it because a circular import error occurs?

Reply:
yes, a lot of Circular Import problems 😓

Reply:
fyi; in the short term you can also do
```python
from typing import TYPE_CHECKING
if TYPE_CHECKING:
   from boa3.internal.model.method import Method
```
type checkers have a different import mechanism